src/redis_info_provider/info_servicer.py

```diff
@@ -9,6 +11,38 @@
 logger = logging.getLogger(__name__)
 
 
+def deprecated_alias(**aliases):
+    """
+    Decorator that renames deprecated parameters names of a function to their new names and then calls the function with
```

Comment:
Nitpick:
```suggestion
    Decorator that renames deprecated parameter names of a function to their new names and then calls the function with
```
Also the `:return:` line is probably unnecessary for a decorator, although I'm not militant about this.